app/models/user.rb

```diff
@@ -1,3 +1,9 @@
 class User < ApplicationRecord
-  # has_many :posts, :comments, :likes
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  def last_3_post
+    posts.last(3)
+  end
```

Comment:
- [ ] We want to fetch a user's **recent** posts, meaning we need to return the posts in order of recency.

  ```
  posts.order(created_at: :desc).limit(3)
  ```